src/pages/LoginPage.ts

```diff
@@ -16,10 +17,30 @@ export default class LoginPage extends BasePage {
         await this.page.context().storageState({ path })
     }
 
-    async signInWith(userName: string, passWord: string): Promise<void> {
-        await this.inputUserName.fill(userName)
-        await this.inputPassword.fill(passWord)
+    async signInWith(userName: UserType): Promise<void> {
+        await this.inputUserName.fill(await this.getUserName(userName))
+        await this.inputPassword.fill(process.env.PASSWORD || '')
```

Comment:
Will this attempt to sign in without a password? Is there a "Fail fast" option, if process.env.PASSWORD null -> exit 1 kinda thing?